var/spack/repos/builtin/packages/typst/package.py

```diff
@@ -18,6 +18,7 @@ class Typst(CargoPackage):
 
     license("Apache-2.0", checked_by="upsj")
 
+    version("0.13.0", commit="8dce676dcd691f75696719e0480cd619829846a9", tag="v0.13.0")
     version("0.12.0", commit="737895d769188f6fc154523e67a9102bc24c872e", tag="v0.12.0")
 
     depends_on("rust@1.81.0:")
```

Comment:
```suggestion
    depends_on("rust@1.80:", type="build", when="@0.13:")
    depends_on("rust@1.77:", type="build")
```

Looking at the `cargo.toml` in the repo it looks like the dependencies on rust should be a bit more like this.

Reply:
I'll check if this causes any compilation issues

Reply:
The 1.80 requirement was deliberate, otherwise we get an error:
```
spack install typst@0.12.0 ^rust@1.78
...
     307    Caused by:
     308      rustc 1.78.0-nightly is not supported by the following packages:
     309        native-tls@0.2.14 requires rustc 1.80.0
     310        native-tls@0.2.14 requires rustc 1.80.0
```

Reply:
Interesting, I wonder why their release is incorrect. 🤷